### PR TITLE
Authentication  Done with Mockbee

### PR DESCRIPTION
**Pull request by m26unkwn**
1. Login and Signup Page Added
2. Account Page Added

### CODE REVIEW

File: src/components/Auth/Signup/index.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+              Login here
+            </Link>
+          </div>
+        </form>
+      </div>
+    </section>
+  );
+};
+
+export default Signup;
```
Comment (deekshithmd) [inline src/components/Auth/Signup/index.js:132]: Code looks clean and good you can proceed to merge.

**Review by deekshithmd — APPROVED** [review]

File: src/App.js [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import "./App.css";
 import Home from "./screens/Home";
-import { Navbar } from "./components";
+import { Navbar, Login, Signup, PrivateRoute, Profile } from "./components";
 import { Routes, Route } from "react-router-dom";
 import Mockman from "mockman-js";
 
 import { useEffect } from "react";
-import { useData } from "./Context/state-context";
+import { useData } from "./Context/stateManage/state-context";
```
Comment (PuneetP16) [inline src/App.js:8]: Suggestion: you could work on coming up with more rational name instead of stateManage.

**Review by PuneetP16 — APPROVED** [review]
Appreciation:  liked the way you have modularized the input components.

It would be better if could visually check the logic on your project live preview.

File: src/components/Auth/Signup/index.js [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            label='Last Name'
+            type='name'
+            placeholder='Doe'
+            onChangeHandler={onChangeHandler}
+            onFocusHandler={onFocusHandler}
+            onChangeParam='SET_LAST_NAME'
+            onFocusParam='SET_LAST_NAME_ERROR'
+            errorMsg='Enter Valid Last Name'
+          />
+          <Input
```
Comment (PuneetP16) [inline src/components/Auth/Signup/index.js:71]: learned new thing, could send error msg to specific input elements.

**Review by PuneetP16 — APPROVED** [review]

File: src/App.js (hunk 2 above)
Comment (m26unkwn) [reply src/App.js:8]: Sure Puneet Will Definitely do something with the name.

**Pull request merged**